pymc3/distributions/transforms.py

```diff
@@ -81,10 +144,32 @@ class Log(ElemwiseTransform):
     name = "log"
 
     def backward(self, x):
-        return tt.exp(x)
+        return_expr = tt.exp(x)
+        
+        # Add a test_value if there is not one already. Useful when operating
+        # with theano.config.compute_test_value = 'off'
+        if not hasattr(return_expr.tag, "test_value"):
+            # Get the numeric object to be used as the test value for x
+            numeric_x = get_single_numeric_obj(x, name='x')
+            # Set the test value for the theano variable to be returned
+            return_expr.tag.test_value = np.exp(numeric_x)
+
+        assert hasattr(return_expr.tag, "test_value")
```

Comment:
Sorry. That was leftover from when I was making changes and debugging. I'll remove it.